plugins/com.gwtplugins.gdt.eclipse.suite/plugin.xml

```diff
@@ -135,35 +135,6 @@
                id="com.gwtplugins.gdt.eclipse.suite.webapp"/>
       </shortcut>
 
-      <!-- GWT Legacy Dev Mode Launch shortcut when using GWT -->
-      <shortcut
-            label="GWT Legacy Development Mode with Jetty"
-            description="Launch legacy GWT development mode with Jetty (Classic Development Mode)"
```

Comment:
I don't think we're ready to remove access to legacy dev mode - but I would be comfortable at least hiding it behind a config setting to start with?

Maybe post 4.0 we could hide this, and leave a config option to re-enable it - if we get support questions asking where to find that setting, we know we need to make it more obvious, but if no one every asks about it again, we could consider actually removing it?

The problem is that there are plenty of deployments out there that rely on all kinds of outdated tools or configurations. GWT keeps legacy dev mode around for IE11 and Edge+trident, both of which are technically deprecated, but still available and supported on some platforms.

IE 11 for example has another EOL date coming up mid-Feb, and is still supported on some OSes https://learn.microsoft.com/en-us/lifecycle/faq/internet-explorer-microsoft-edge#what-is-the-lifecycle-policy-for-internet-explorer-

Reply:
What about adding a suffix of (Deprecated) at the end.  That will help encourage existing users to get on a path for an upgrade and also discourage new users from choosing an older way of doing things.  Thoughts?

Reply:
Adding the suffix sounds good. I will create a new pull request, because this one contains to many changes :-(